### PR TITLE
feat(docs): update how we run checks with the new retries flow [sc-14873]

### PR DESCRIPTION
**Pull request by adamwardecki**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

Changed the diagram and steps description based on the new retries flow.

The PR will need to wait for Check Retries GA to be merged.

## Screenshots

image

### CODE REVIEW

File: site/content/docs/monitoring/_index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You can select one or more data center locations to run your checks from. We adv
 There are two reasons for this:
 
 1. Redundancy: we might have an issue in one location, but not the other. 
-2. Double checking: if your check fails and you have "double check" enabled we will retry the check from the other location.
+2. Double checking: if your check fails and you have "double-check" enabled we will retry the check from the other location.
```
Comment (miliberlin) [inline site/content/docs/monitoring/_index.md:22]:
```suggestion
2. Double checking: if your check fails and you have "double-check" enabled we will retry the check from another location.
```

**Review by miliberlin — CHANGES_REQUESTED** [review]

**Review by miliberlin — DISMISSED** [review]

File: site/content/docs/monitoring/_index.md [hunk 2]
```diff
@@ -30,11 +30,13 @@ A picture is a thousand words:
...
-3. The check is executed. If the check fails and "double check" is enabled, the check is retried once in a different location.
-The other location is picked, at random, from all the configured locations. If only one location has been selected, then the other location is picked at random from all available locations. Setup scripts are executed again during the retry.
+3. The check is executed.
 4. If the check is an API check and has a [teardown script](/docs/api-checks/setup-teardown-scripts/), the teardown script is executed.
 Teardown scripts are run *before* any assertions are validated.
-5. The result is stored in our central database and any alerts are sent where applicable.
+5. The result is stored in our central database.
+6. If the check fails and “double-check” is enabled, the process starts over from a different location. The other location is picked, at random, from all the configured locations. If only one location has been selected, then the other location is picked at random from all available locations. Any setup & teardown scripts are run again as part of the process.
+7. When applicable, alerts are send out in requested channels.
```
Comment (jan-osch) [inline site/content/docs/monitoring/_index.md:38]: I think we need to explain here when the whole sequence finishes.
* check passed, or the final attempt was executed
* only then we do send alerts (not for initial attempts)

Comment (adamwardecki) [reply site/content/docs/monitoring/_index.md]: @jan-osch Done! Lemme know if it's good.

File: site/content/docs/monitoring/_index.md [hunk 3]
```diff
@@ -30,11 +30,11 @@ A picture is a thousand words:
...
+3. The check is executed.
 4. If the check is an API check and has a [teardown script](/docs/api-checks/setup-teardown-scripts/), the teardown script is executed.
 Teardown scripts are run *before* any assertions are validated.
-5. The result is stored in our central database and any alerts are sent where applicable.
-
+5. The result is stored in our central database.
+6. If the check fails and “double-check” is enabled, the process starts over from a different location. The other location is picked, at random, from all the configured locations. If only one location has been selected, then the other location is picked at random from all available locations. Any setup & teardown scripts are run again as part of the process.
+7. Alerts are sent out in requested channels when the sequence is complete. It's considered complete when the check run was successful or the final attempt was executed. We will send alerts only if the final attempt has failed (no alerts sent for the initial attempts)
```
Comment (jan-osch) [inline site/content/docs/monitoring/_index.md:38]:
```suggestion
7. When the sequence is complete, we send alerts to the channels as requested. A sequence is considered complete if either the check run was successful or the final attempt was executed. Please note that we will only send alerts if the final attempt has failed. No alerts will be sent for the initial attempts.
```

Asked ChatGPT to improve the text.

**Review by jan-osch — DISMISSED** [review]
LGTM. I left an optional suggestion to make it more readable from ChatGPT

**Review by drakirnosslin — DISMISSED** [review]

**Pull request merged**